Mainframe3270/x3270.py

```diff
@@ -21,12 +21,16 @@ def __init__(
         wait_time: timedelta,
         wait_time_after_write: timedelta,
         img_folder: str,
+        height: int = 24,
```

Comment:
same as the comment before, it would be better to introduce just one new import parameter: "model" that accepts a string that can be one of the x3270 emulators.